lang/cy.json

```diff
@@ -85,49 +85,49 @@
     "addressLine3Label": "Llinell cyfeiriad 3",
     "addressLine2OfYourAddress": "Welsh",
     "townOrCity": "Tref neu ddinas",
-    "yourTownOrCity": "Welsh",
+    "yourTownOrCity": "Eich tref neu ddinas",
     "couldNotLookupPostcode": "Methu chwilio’r cod post",
-    "invalidPostcode": "welsh",
-    "aPostcode": "welsh",
-    "yourPostcode": "Welsh",
-    "anAddressFromTheList": "welsh",
-    "yourAddressFromTheList": "Welsh",
+    "invalidPostcode": "cod post dilys",
+    "aPostcode": "cod post",
+    "yourPostcode": "eich cod post",
+    "anAddressFromTheList": "cyfeiriad o'r rhestr",
+    "yourAddressFromTheList": "eich cyfeiriad o'r rhestr",
     "noAddressesFound": "Welsh",
     "noYourAddressesFound": "Welsh",
-    "whoIsThisLpaFor": "Ar gyfer pwy mae’r LPA hwn?",
+    "whoIsThisLpaFor": "Ar gyfer pwy mae’r LPA hon?",
     "whoTheLpaIsFor": "Welsh",
-    "forMe": "I mi",
-    "forSomeoneElse": "I rywun arall",
-    "imHelping": "Rwy’n helpu rhywun arall i lenwi’r ffurflen LPA",
+    "forMe": "Fi fy hun",
+    "forSomeoneElse": "Rhywun arall",
+    "imHelping": "Rydw i’n helpu rhywun arall i lenwi’r ffurflen LPA.",
     "email": "Cyfeiriad e-bost",
-    "emailOptional": "Welsh",
+    "emailOptional": "Cyfeiriad e-bost (dewisol)",
     "phone": "Ffôn",
     "post": "Post",
-    "whichTypeOfLpaToMake": "Pa fath o LPA ydych chi am ei wneud?",
-    "whichTypeOfLpaToMakeContent": "<p class=\"govuk-body\">Mae 2 fath o LPA. Gallwch wneud LPA sy’n cynnwys penderfyniadau am eich:</p><ul class=\"govuk-list govuk-list--bullet\"><li>arian, cyllid ac unrhyw eiddo rydych yn berchen</li><li>ofal iechyd, triniaeth feddygol a lles personol</li></ul><p class=\"govuk-body\">Gallwch gael un neu’r ddau fath o LPA.</p><p class=\"govuk-body\">Nid oes rhaid iddynt gael eu gwneud ar yr un pryd. Gallwch ailddefnyddio unrhyw wybodaeth i wneud LPAs eraill yn hwyrach.</p><p class=\"govuk-body\">Gallwch ddewis gwahanol bobl i fod yn atwrneiod i chi ar gyfer pob LPA.</p><p class=\"govuk-body\"><a class=\"govuk-link\" href=\"#\">Dysgwch fwy am y 2 fath o LPAs</a></p>",
+    "whichTypeOfLpaToMake": "Pa fath o LPA ydych chi eisiau ei gwneud?",
+    "whichTypeOfLpaToMakeContent": "<p class=\"govuk-body\">Ceir 2 fath o LPA. Gallwch wneud LPA sy'n cynnwys penderfyniadau ynglŷn â’ch:</p><ul class=\"govuk-list govuk-list--bullet\"><li>arian, cyllid ac unrhyw eiddo sydd gennych chi</li><li>gofal iechyd, triniaeth feddygol a lles personol</li></ul><p class=\"govuk-body\">Gallwch gael un neu'r ddau fath o LPA.</p><p class=\"govuk-body\">Nid oes rhaid eu gwneud ar yr un pryd a gallwch ailddefnyddio unrhyw wybodaeth a roddwch i wneud LPA arall nes ymlaen.</p><p class=\"govuk-body\">Gallwch ddewis gwahanol bobl i fod yn atwrneiod i chi ar gyfer pob LPA.</p><p class=\"govuk-body\"><a class=\"govuk-link\" href=\"#\">Darllenwch fwy am y 2 fath o LPA.</a></p>",
```

Comment:
Is the change of `sy’n` -> `sy'n` (and other instances of non-curly apostrophes) intentional? This also suggests our lang paths are broken in `mlpa/main_test.go`.

Reply:
ah yes, those tests have the wrong paths now